### ISSUE TITLE
Invalid link between entites

### ISSUE THREAD

**Issue by atomfrede** [jhipster/jhipster-vuejs#498]
##### **Overview of the issue**

The link between entities is not correct as the parameter name does not have wrong casing.

##### **Motivation for or Use Case**

The link to the other entity does not work out of box, but it should

![image](https://user-images.githubusercontent.com/203401/68895441-900c4c00-0729-11ea-81a2-ef5b12fe5d6f.png)

##### **Reproduce the error**

Create a default application and import following JDL:

```
entity BookCopy {
    signature String required unique,
	title String required,
}

entity Borrow {
    borrowedAt LocalDate required,
}

relationship ManyToOne {
    Borrow{bookcopy(signature)} to BookCopy
    Borrow{user(login)} to User
}
```

##### **Related issues**

None

##### **Suggest a Fix**

The variable `otherEntityName` seems to be incorrect. Either use another one (`otherEntityAngularName` is also not correct) not sure if there is already a fitting variable. Or  

##### **JHipster Version(s)**

Both main generator and blueprint using latest master

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.mycompany.myapp",
      "nativeLanguage": "de"
    },
    "jhipsterVersion": "6.5.0",
    "applicationType": "monolith",
    "baseName": "jhth",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "caffeine",
    "enableHibernateCache": true,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "postgresql",
    "prodDatabaseType": "postgresql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "gradle",
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "bXktc2VjcmV0LXRva2VuLXRvLWNoYW5nZS1pbi1wcm9kdWN0aW9uLWFuZC10by1rZWVwLWluLWEtc2VjdXJlLXBsYWNl",
    "embeddableLaunchScript": false,
    "testFrameworks": [
      "protractor"
    ],
    "jhiPrefix": "jhi",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "otherModules": [
      {
        "name": "generator-jhipster-vuejs",
        "version": "1.2.0"
      }
    ],
    "enableTranslation": true,
    "clientPackageManager": "npm",
    "nativeLanguage": "de",
    "languages": [
      "de",
      "en"
    ],
    "blueprints": [
      {
        "name": "generator-jhipster-vuejs",
        "version": "1.2.0"
      }
    ]
  },
  "generator-jhipster-vuejs": {
    "jhipsterVersion": "6.4.1",
    "applicationType": "monolith",
    "baseName": "jhth",
    "useSass": true,
    "clientPackageManager": "npm",
    "clientFramework": "vue",
    "clientTheme": "none",
    "clientThemeVariant": ""
  }
}
</pre>
</details>

##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>
entity BookCopy {
  signature String required unique,
  title String required
}
entity Borrow {
  borrowedAt LocalDate required
}
relationship ManyToOne {
  Borrow{bookcopy(signature)} to BookCopy,
  Borrow{user(login)} to User
}

</pre>
</details>

##### **Environment and Tools**

openjdk version "1.8.0_232"
OpenJDK Runtime Environment (build 1.8.0_232-b09)
OpenJDK 64-Bit Server VM (build 25.232-b09, mixed mode)

git version 2.24.0

node: v10.16.3

npm: 6.13.0

yeoman: 3.1.0

yarn: 1.19.1

Docker version 19.03.4-ce, build 9013bf583a

docker-compose version 1.24.1, build unknown

INFO! Congratulations, JHipster execution is complete!

**Comment by pascalgrimaud** [comment]
are you sure you're not using jhipster v6.5.0 with problem about lazy ?

**Comment by atomfrede** [comment]
Yes. The is is there. If I change 
```
<router-link :to="{name: 'BookCopyView', params: {bookcopyId: borrow.bookcopy.id}}">{{borrow.bookcopy.signature}}</router-link>
```
to
```
<router-link :to="{name: 'BookCopyView', params: {bookCopyId: borrow.bookcopy.id}}">{{borrow.bookcopy.signature}}</router-link>
```
the link is fine. Mind the param name which must be `bookCopyId` but is generated as `bookcopyId`

**Comment by hdurix** [comment]
Hello @atomfrede, I just tried to reproduce this issue but I was not able to.

Here what I've got:

```
<router-link :to="{name: 'BookCopyView', params: {bookCopyId: borrow.bookcopy.id}}">{{borrow.bookcopy.signature}}</router-link>
```

I think this has been fixed in https://github.com/jhipster/jhipster-vuejs/pull/488 by @ctamisier.

You opened this ticked after the PR was merged but maybe you were not using the latest master version? I let you close the ticket if you think this has been fixed.
